docker/lr-cartographer/wdl_and_json/cartographer.wdl

```diff
@@ -0,0 +1,310 @@
+version 1.0
```

Comment:
I kept a `wdl_and_json` folder around for doing development on this tool and I thought it would be useful to have it in the development area so that things could be tested individually.  This is actually a holdover from developing this tool in the other `lrma` repo.   

I'm not opposed to removing the folder alltogether.  @kvg What do you think?